src/Lykke.Exchange.Api.MarketData/Program.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Lykke.Common;
+using Microsoft.AspNetCore.Hosting;
+
+namespace Lykke.Exchange.Api.MarketData
+{
+    [UsedImplicitly]
+    public class Program
```

Comment:
It's better use Lykke.SDK, like
        public static async Task Main(string[] args)
        {
#if DEBUG
            await LykkeStarter.Start<Startup>(true);
#else
            await LykkeStarter.Start<Startup>(false);
#endif
        }